src/extension.ts

```diff
@@ -366,7 +367,7 @@ async function setupAuthProvider(): Promise<vscode.Disposable[]> {
   // Send an Identify event to Segment with the session info if available
   if (cloudSession) {
     sendTelemetryIdentifyEvent({
-      eventName: UserEvent.ActivatedWithSession,
+      eventName: UserEvent.ExtensionActivation,
```

Comment:
Is this really the right key we want? It seems to me that ccloud login is entirely orthogonal to extension activation and warrants its own key.

Reply:
And now that I look at it more closely, we could also change `SignedIn` to `CCloudAuth` and separate sign-in vs sign-out events...